README.md

```diff
@@ -1 +1,24 @@
-# angular-integrate-http-lab
\ No newline at end of file
+# Integrate $http into CRUD service
+
+## Objectives
+
+- Use a backend API to fetch data
+- Create public Service methods to match API intention
+
+## Instructions
+
+Let's create a service to query a stubbed API. To do this, we need `stubby` as a command line program.
+
+We can do this with
+
+```bash
+sudo npm install -g stubby
+```
+
+Then, run `stubby -d config.yml` in your command line.
+
+Now, we've got our mocked backend running. This has two endpoints under the url `/rest/user` - we can issue both GET and POST requests to it.
+
+Create a service that queries this endpoint, and display the results in the view.
```

Comment:
Instead of having them display information in a view, is there a way to use Jasmine tests to make sure that the return values are what they should be? This would be easier for the user, as they could just make sure that their service is returning the correct information outside the context of a view.